release/RELEASE.md

```diff
@@ -130,23 +138,37 @@ our luck with a random new one.
       Verify that the scenario result appears within 30 seconds.
    1. Add `+` at the end of line 26 after `"Alice"` and verify that you get an error.
 
-1. **[STABLE]** If there are no issues, the release can be made public.
-   Go to [the releases page](https://github.com/digital-asset/daml/releases)
-   and click on the `Edit` button for the new release. Combine the release
-   notes from `docs/source/support/release-notes.rst` for all releases since the
-   last public release, convert them from RST to Markdown format, and insert them
-   in the textbox. Uncheck the `This is a pre-release` checkbox at the bottom and
-   click `Update release`.
+1. On your PR, add the comment
+
+   > Manual tests passed on Windows.
```

Comment:
I guess those separate messages are a preparation for maybe doing more lenient tests in the future for snapshot releases?

Reply:
My goal was just to clarify the process that we already have. We've been supposed to put those messages on release PRs ever since we added the Standard-Change label.